qa/suites/upgrade/octopus-x/parallel/1-tasks.yaml

```diff
@@ -16,7 +16,6 @@ tasks:
         osd_class_default_list: "*"
     # deploy additional mons the "old" (octopus) way
     add_mons_via_daemon_add: true
-    avoid_pacific_features: true
```

Comment:
We don't need to test octopus any longer, just pacific-x and quincy-x.

Reply:
removed `octopus-x`